distributedID/blockchain-explorer
# Adding the payload field to be displyed. Only UUID is being displayed.

### Ref:
#25 

### Description:
As it says on the tin.

### Blocking:
None

### Documentation:
None

### Testing:
1. Use `npm start` to start the explorer.
1. Visit `transactions` tab on `localhost:11000`. 
1. Click on any `ENDORSER_TRANSACTION`. 
1. You should be able to see the `Payload: 25fe...` in the dialog box.
